samples/apps/copilot-chat-app/webapp/README.md

```diff
@@ -5,6 +5,59 @@ production deployments.
 
 See [../README.md](../README.md) for complete instructions on setting up and running the application.
 
+## How to use HTTPS for local development
+If you need to run Copilot Chat with HTTPS, you need to create a certificate and sign it with a Certificate Authority (CA) that is trusted locally by your device and browser. You can do that easily with [mkcert](https://github.com/FiloSottile/mkcert#installation).
+
+1. From an elevated shell, run
+    ```
+    choco install mkcert
```

Comment:
Introducing `choco` adds another Windows dependency? Users should be able to use this sample on WSL2/Linux (e.g. devcontainer) or on macOS as well in my opinion.